doc/getting-started/apl-nuc.rst

```diff
@@ -120,13 +120,14 @@ partition. Follow these steps:
 
    .. code-block:: none
 
-      # mount /dev/sda1 /mnt
+      $ sudo mount /dev/sda1 /mnt
 
-      # ls -1 /mnt/EFI/org.clearlinux
+      $ sudo ls -1 /mnt/EFI/org.clearlinux
       bootloaderx64.efi
       kernel-org.clearlinux.native.4.19.1-654
       kernel-org.clearlinux.iot-lts2018-sos.4.19.0-19
       kernel-org.clearlinux.iot-lts2018.4.19.0-19
+      kernel-org.clearlinux.pk414-sos.4.14.74-115
```

Comment:
I guess this is installed by the `service-os` bundle, isn't it?

@miguelinux , are we still using that `sos` kernel on any platform?